spec/dummy/.controlplane/controlplane.yml

```diff
@@ -11,7 +11,7 @@ aliases:
       - postgres
 
 apps:
-  dummy-test:
+  dummy-test-{GLOBAL_IDENTIFIER}:
```

Comment:
We now need to add this placeholder and replace it when running the tests to ensure that everything works properly. We replace it in `dummy_app_setup.rb`.